### PR TITLE
Test/game factory/goals at home and away n (not sure issue number)

### PR DESCRIPTION
**Pull request by NicoShanstrom**
tests:
created tests for goals_at_away(team_id and goals_at_home(team_id) at the the of game_factory_spec file. 
Getting the error when running file, maybe this is already known?
LoadError:
  cannot load such file -- simplecov
# ./spec/spec_helper.rb:1:in `<top (required)>'
# ./spec/game_factory_spec.rb:1:in `<top (required)>'
No examples found.

### CODE REVIEW

File: spec/game_factory_spec.rb [hunk 1]
```diff
@@ -124,4 +124,32 @@
...
+
+      expect(@game_factory.goals_at_away(3)).to be_a(Integer)
+    end
+
+    it 'returns the number goals of away team from a given team_id' do
+      @game_factory.create_games
+
+      expect(@game_factory.goals_at_away(3)).to eq(5)
+    end
+  end
```
Comment (JackCSweeney) [inline spec/game_factory_spec.rb:154]: Looks great! Thank you for jumping on this first thing today!

File: spec/stat_tracker_calculator_spec.rb [hunk 2]
```diff
@@ -251,4 +251,3 @@
         end
     end
 end
-end
```
Comment (JackCSweeney) [inline spec/stat_tracker_calculator_spec.rb]: I believe this end that was removed in the stat_tracker_calculator_spec file was actually needed here. There is just some spacing issues in the file still which is why it looks like its an extra one. Can you add it back in for us Nico?

**Review by JackCSweeney — CHANGES_REQUESTED** [review]

File: spec/stat_tracker_calculator_spec.rb (hunk 2 above)
Comment (NicoShanstrom) [reply spec/stat_tracker_calculator_spec.rb]: done, added 'end' to line 254. I must have been on main and not this branch when looking at it before. Also, ran bundle_install like you suggested and that ran as needed.

Comment (JackCSweeney) [reply spec/stat_tracker_calculator_spec.rb]: Perfect! Thank you sir!

**Pull request merged**